resume/src/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './App.css';
+import Header from './components/Header';
+import Sidebar from './components/Sidebar';
+import Main from './components/Main';
+import Footer from './components/Footer';
+import data from './data/data.json';
+
+function App() {
+  return (
+    <div className="App">
+      <Header
+        basics={data.basics}
+      />
+      <div className="middle-container">
+        <Sidebar
+          basics={data.basics}
+          skills={data.skills}
+        />
+        <Main
+          work={data.work}
+          volunteer={data.volunteer}
+          education={data.education}
+        />
+      </div>
+      <Footer
```

Comment:
I think it would more sense to have sister components for each of the footer sections, semantically they don't necessarily fit into a footer

Reply:
I completely agree! I really ran out of time at the end and started to rush so didn't get a chance to organize this section better. Thanks for the feedback!